src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Fluent/Themes/Fluent.HC.xaml

```diff
@@ -2093,6 +2093,7 @@
     <Setter Property="Border.CornerRadius" Value="{DynamicResource ControlCornerRadius}" />
     <Setter Property="SnapsToDevicePixels" Value="True" />
     <Setter Property="OverridesDefaultStyle" Value="True" />
+    <Setter Property="KeyboardNavigation.TabNavigation" Value="Local" />
```

Comment:
Just for my learning purposes - 
How did this fix the accessibility issue?

Reply:
Earlier this was set to Once by default, due to which only one child of the DatePicker control would get the navigation and then skip to other control. Now it will go to each of the siblings. Since there are two, textbox and the button, it goes to each one of them.

Reply:
Here's the docuement for your reference : https://learn.microsoft.com/en-us/dotnet/api/system.windows.input.keyboardnavigationmode?view=windowsdesktop-8.0